ingestion_tools/scripts/importers/tiltseries.py

```diff
@@ -92,6 +97,10 @@ def import_metadata(self) -> None:
         merge_data["frames_count"] = self.get_frames_count()
         base_metadata = self.get_base_metadata()
         merge_data["pixel_spacing"] = self.get_pixel_spacing()
+        psp_generator = PerSectionParameterGenerator(self.config, {**self.parents, "tiltseries": self})
```

Comment:
Just a headsup that if PerSectionParameterGenerator was a class property and called from that property instead of referenced directly here, it could be overridden in tests easily. It looks like you don't strictly *need* to override this in your tests, but it's another tool you could use to inject dependencies better.